src/Login/components/LoginComponent.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Login = (
+    {
+        handleSubmit,
+        onChange,
+        email,
+        password,
+        errors
+    }
+) => {
+    return (
+
+        <div className="container">
+            <h1>Login</h1>
+            <div className="myform">
+                <form >
+                    <div className="form-input"  id="errormessage">
+                        {/* <label>Email:</label> */}
+                        <input type="text" id="userEmail" name="email" placeholder="Email" value={email} name="email" onChange={onChange} />
+                        {
+                            errors.userEmail && (<div style={{ color: 'red', textAlign:"center"}}>{errors.userEmail}</div>)
```

Comment:
Why are you using inline styles?

Reply:
it was the only easlier way l would style and display the error messages as recommended by the PO's feedback.